pkg/statistics/handle/types/interfaces.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package util
+package types
```

Comment:
```suggestion
package handleimpl
```

It is better to call ```handleimpl```.